packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -257,6 +267,7 @@ public function import_user( $data ) {
 		 * @param array $userdata Raw data imported for the user.
 		 */
 		do_action( 'wxr_importer_processed_user', $user_id, $userdata );
+		// $this->topological_sorter->map_entity( 'user', $userdata, $user_id );
```

Comment:
Should this be uncommented? Or is it a placeholder for later? Then let's document inline how will we know to uncomment it.

Reply:
It is commented out because I'm not sure what we do with `wp_insert_user`. If we will reassign, rewrite or possibly remap the user of the wxr with the preexisting one, like in other importers.